public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -313,3 +316,14 @@ function parseHistogramLabel(le: string): number {
   }
   return Number(le);
 }
+
+function parseSampleValue(value: string): number {
```

Comment:
Thanks for your contribution @KonanMentor this looks good. Can you please use your function in places where parseHistogramLabel used as well?

Reply:
Done, please check